frb_codegen/src/utils.rs

```diff
@@ -42,6 +42,72 @@ where
         .collect::<Vec<_>>()
 }
 
+const DART_KEYWORDS: [&str; 63] = [
```

Comment:
uber-nit: would be great to add a comment where this list comes from (e.g. dart website?)